### DISCUSSION TITLE
Welcome to ReVot Discussions!

### DISCUSSION THREAD

**Discussion by c0sm0void** [minnonymous/ReVot#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jayanth-M0625** [comment]
Are there any issues specifically you would like us to work on or begin from?

**Reply by c0sm0void** [reply]
Soon will update the requirements. Please wait for a moment.

**Reply by c0sm0void** [reply]
@Jayanth-M0625 you can create new or ask for the existing issue to work on.
